internal/sirius/get_document_types.go

```diff
@@ -4,7 +4,22 @@ import (
 	"github.com/ministryofjustice/opg-sirius-supervision-deputy-hub/internal/model"
 )
 
-func (c *Client) GetDocumentTypes(ctx Context) ([]model.RefData, error) {
+func (c *Client) GetDocumentTypes(ctx Context, deputyType string) ([]model.RefData, error) {
 	documentTypes, err := c.getRefData(ctx, "?filter=noteType:deputy")
-	return documentTypes, err
+	var documentTypesFilteredByDeputyType []model.RefData
+	documentTypesFilteredByDeputyType = filterDocTypeByDeputyType(documentTypes, deputyType)
+	return documentTypesFilteredByDeputyType, err
+}
+
+func filterDocTypeByDeputyType(DocumentTypes []model.RefData, DeputyType string) []model.RefData {
+	var deputySpecificDocTypes []model.RefData
+	for _, v := range DocumentTypes {
+		if v.Label != "Indemnity insurance" && DeputyType == "PA" {
+			deputySpecificDocTypes = append(deputySpecificDocTypes, v)
+		}
+		if v.Label != "Catch-up call" && DeputyType == "PRO" {
```

Comment:
This is a minor adjustment, but would you consider checking v.Handle rather than v.Label? Just because the handle is less likely to be updated in the future etc

Reply:
done